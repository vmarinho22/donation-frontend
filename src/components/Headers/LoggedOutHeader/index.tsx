import { Button } from "@/components/ui/button";
import { useTranslations } from "next-intl";
import Link from "next/link";

export default function LoggedOutHeader() {
  const t = useTranslations('buttons');

  return (
    <header className='h-22 mb-8 flex w-full flex-nowrap items-center justify-between px-8 py-3'>
        <h1 className='text-2xl font-bold'>Donation System</h1>
        <Button
          variant='outline'
          asChild
          className='border-bloodRed-500 hover:bg-bloodRed-500 hover:text-white'
        >
          <Link href='/login'>{t('signin')}</Link>
        </Button>
      </header>
  );
}
