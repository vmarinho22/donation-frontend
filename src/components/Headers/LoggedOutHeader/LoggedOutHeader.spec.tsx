import {render, screen} from '@testing-library/react';
import TestWithLocaleWrapper from '@/components/test/TestWIthLocaleWrapper';
import LoggedOutHeader from '.';

describe('LoggedOutHeader', () => {

  it('should render the component', () => {
    render(
        <LoggedOutHeader />
    );

    expect(screen.getByText(`Donation System`)).toMatchSnapshot();
  });

});