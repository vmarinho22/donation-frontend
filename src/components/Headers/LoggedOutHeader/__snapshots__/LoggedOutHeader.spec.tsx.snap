// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoggedOutHeader should render the component 1`] = `
<h1
  class="text-2xl font-bold"
>
  Donation System
</h1>
`;
